### PR TITLE
did stuff....

### PR DESCRIPTION
**Pull request by jacobjfoley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,23 +9,14 @@ In the project directory, you can run:
...
 ### `yarn build`
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+will build the project, Ready to deployyyyy
```
Comment (elhamsdtn) [inline README.md:19]: typo here?

File: package.json [hunk 2]
```diff
@@ -3,14 +3,16 @@
...
     "axios": "^0.19.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.1"
   },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2"
+  },
```
Comment (elhamsdtn) [inline package.json:15]: can you pls add to your PR comment that you did this and explain the reason for that

File: src/components/App/index.js [hunk 3]
```diff
@@ -81,6 +81,9 @@ class App extends Component {
         </form>
         <h4>{this.state.searchMessage}</h4>
         {this.state.bookIsLoading && <Loading />}
+        {this.state.bookIsLoading && <Loading />}
```
Comment (elhamsdtn) [inline src/components/App/index.js:84]: any specific reason that there is repetition here?

File: src/components/Loading/index.js [hunk 4]
```diff
@@ -2,6 +2,11 @@ import React from "react";
...
 
 function Loading() {
-  return <h4 className={styles.loading}>"Loading ..."</h4>;
+  return <h4 className={styles.loading}>"Loadddding ..."</h4>;
 }
+
+
+
+
+
```
Comment (elhamsdtn) [inline src/components/Loading/index.js:11]: have you prettier and Eslint installed? if you do that it will fix the extra line for you

**Review by elhamsdtn — CHANGES_REQUESTED** [review]
There are a couple of formatting and typos and also would be helpful to be a bit more clear for the logic of changes

File: README.md (hunk 1 above)
Comment (jacobjfoley) [reply README.md:19]: I like it better this way.

File: package.json (hunk 2 above)
Comment (jacobjfoley) [reply package.json:15]: I think it's pretty clear already isn't it?

File: src/components/App/index.js (hunk 3 above)
Comment (jacobjfoley) [reply src/components/App/index.js:84]: Yes.

File: src/components/Loading/index.js (hunk 4 above)
Comment (jacobjfoley) [reply src/components/Loading/index.js:11]: I don't have time to install them.

**Pull request closed without merging**